mmdet/apis/train.py

```diff
@@ -131,6 +131,7 @@ def train_detector(model,
             dist=distributed,
             shuffle=False)
         eval_cfg = cfg.get('evaluation', {})
+        eval_cfg['test_cfg'] = cfg.get('test_cfg', {})
```

Comment:
So, that's it?

Reply:
I should update config too in the other PR too but yes. It's pretty elegant, isn't it?